org.ekstep.questionset-1.0/editor/style/style.css

```diff
@@ -14,4 +14,18 @@
     -ms-flex-flow: column;
     -o-flex-flow: column;
     flex-flow: column;
+}
+
+.qs-scrollbar::-webkit-scrollbar {
```

Comment:
@itsvick is this changes verified in all browsers?

Reply:
No, as its specific to question-set plugin only, and added custom class for it.
@pallakartheekreddy

Reply:
After increasing width of the scroll bar -
1. On chrome: 
![scrollbar-chrome](https://user-images.githubusercontent.com/20536805/52463743-2c536200-2b9e-11e9-9e0f-95de0db806f5.png)
2. On Firefox:
![firefox-scrollbar](https://user-images.githubusercontent.com/20536805/52463759-3d03d800-2b9e-11e9-99b6-a8bfaeb12d0e.png)